magic/core/src/main/java/link/magic/android/core/relayer/message/Announcement.kt

```diff
@@ -0,0 +1,16 @@
+package link.magic.android.core.relayer.message
+
+data class AnnouncementMessage(
+    val msgType: String,
+    val response: Response
+)
+
+data class Response(
```

Comment:
This is redundant with Response from web3j

Reply:
This is what I get for rushing 😮‍💨